library/junos_install_config

```diff
@@ -275,7 +284,27 @@ def _load_via_netconf(module):
         elif overwrite is False:
             load_args['merge'] = True
         cu.load(**load_args)
-    except (ValueError, ConfigLoadError, Exception) as err:
+    # If only "statement not found" warning messages are raised and 
+    # commit_onwarnings is set to "True", commit proceeds, otherwise fails. 
+    except ConfigLoadError as err:
+        severity = list(error['severity'] for error in err.errs)
+        messages = list(error['message'] for error in err.errs)
+
+        if not ((len(set(severity)) <=1 and severity[0] == 'warning') and
+                (len(set(messages)) <=1 and messages[0] == 'statement not found') and
+                commit_onwarnings):
+            msg = "Unable to load config: {0}".format(err)
+            logging.error(msg)
+            try:
+                logging.info("unlocking")
+                cu.unlock()
+            except UnlockError as err:
+                logging.error("Unable to unlock config: {0}".format(err))
+            dev.close()
+            module.fail_json(msg=msg)
+        else:
+            logging.info("Warnings 'statement not found' were raised but ignored, proceeding with commit.")
```

Comment:
Change from `"Warnings 'statement not found' were raised but ignored,` to `"'statement not found' warnings were raised but ignored,`